strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/config/StorageMultipartConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.carlspring.strongbox.config;
+
+import org.carlspring.strongbox.config.util.UpdatableMultipartConfigElement;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.web.servlet.MultipartProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+
+import javax.inject.Inject;
+import javax.servlet.MultipartConfigElement;
+
+@EnableConfigurationProperties(MultipartProperties.class)
+public class StorageMultipartConfig
```

Comment:
You need to add this annotation so Spring can scan the Bean created inside.
```suggestion
@Configuration
@EnableConfigurationProperties(MultipartProperties.class)
public class StorageMultipartConfig
```

And, also, in `org.carlspring.strongbox.config.StorageApiConfig`, you need to add `"org.carlspring.strongbox.config"` to the `@ComponentScan` list so Spring can read the components (`StorageMultipartConfig`) on this package.